RefrigeratorSaraFarkash/RefrigeratorSaraFarkash/CRefrigerator.cs

```diff
@@ -0,0 +1,334 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace RefrigeratorSaraFarkash
+{
+    class CRefrigerator : IComparable<CRefrigerator>
+    {
+        private static int countObj = 0;
+
+        private int id;
+
+        public int Id
+        {
+            get { return id; }
+            //set { id = countObj++; }
+        }
+
+        private string model;
+
+        public string Model
+        {
+            get { return model; }
+            set
+            {
+                if (!value.Equals(""))
+                    model = value;
+                else
+                    throw new ArithmeticException("Illegal model, model must be contain string");
+            }
+        }
+
+        private string color;
+
+        public string Color
+        {
+            get { return color; }
+            set
+            {
+                if (!value.Equals(""))
+                    color = value;
+                else
+                    throw new ArithmeticException("Illegal color, color must be contain string");
+            }
+        }
+        private int numberOfShelves;
+
+        public int NumberOfShelves
+        {
+            get { return numberOfShelves; }
+            set
+            {
+                if (value >= 0)
+                    numberOfShelves = value;
+                else
+                    throw new ArithmeticException("Illegal numbershelves, numbershelves must be greater or equal zero");
+            }
+        }
+
+        public List<CShelf> shelfs;
+
+        public List<CShelf> Shelfs
+
+        {
+            get { return shelfs; }
+            //set
+            //{
+            //    if (!(value is null))
+            //    {
+            //        shelfs = value;
+            //    }
+            //    else
+            //    {
+            //        shelfs = new List<CShelf>();
+            //    }
+            //}
+        }
+
+        public CRefrigerator()
+        {
+            id = countObj++;
+            shelfs = new List<CShelf>();
+        }
+
+        public void AddShelfs(CShelf shelf)
```

Comment:
There are no shelves without a refrigerator why are you creating them outside of it?